package.json

```diff
@@ -43,5 +43,9 @@
     "kind-of": "^6.0.3",
     "minimist": "^1.2.2",
     "sanitize-html": "~1.20.0"
+  },
+  "resolutions": {
```

Comment:
can you merge this with the above section?

Reply:
lol, I didn't even see that - no problem